init_db.sql

```diff
@@ -0,0 +1,17 @@
+CREATE DATABASE `manufacturer_db`;
+
+CREATE TABLE `manufacturers` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `country` varchar(45) NOT NULL,
+  `is_deleted` tinyint NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
+
+CREATE TABLE `drivers` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(45) NOT NULL,
+  `licenseNumber` varchar(45) NOT NULL,
```

Comment:
```suggestion
  `license_number` varchar(45) NOT NULL,
```
it is better NOT to use camelCase for column names

Reply:
changed